packages/@aws-cdk/aws-kinesis/lib/stream.ts

```diff
@@ -752,9 +755,15 @@ export class Stream extends StreamBase {
       physicalName: props.streamName,
     });
 
-    const shardCount = props.shardCount || 1;
+    let shardCount = props.shardCount;
+    let streamMode = props.streamMode ?? StreamMode.PROVISIONED;
```

Comment:
Can be constant

Reply:
In this commit, you are making two changes, ie. the fix and the default to PROVISIONED capacity. You can include that in the commit details.

Reply:
I've updated the description. Hopefully that's a bit clearer.